PhysicsTools/PatAlgos/src/MuonMvaIDEstimator.cc

```diff
@@ -0,0 +1,108 @@
+#include "PhysicsTools/PatAlgos/interface/MuonMvaIDEstimator.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+//#include "FWCore/Framework/interface/stream/EDProducer.h"
+//#include "FWCore/Utilities/interface/StreamID.h"
+//#include "FWCore/Framework/interface/stream/EDAnalyzer.h"
+
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+#include "CommonTools/MVAUtils/interface/GBRForestTools.h"
+#include "DataFormats/Candidate/interface/Candidate.h"
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/MuonReco/interface/MuonSelectors.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "DataFormats/JetReco/interface/PFJet.h"
+#include "DataFormats/JetReco/interface/PFJetCollection.h"
+#include "DataFormats/PatCandidates/interface/Muon.h"
+#include "JetMETCorrections/JetCorrector/interface/JetCorrector.h"
```

Comment:
is this actually used? please double check that all includes are in fact required, in other words, that you have a "minimal set" of includes.

Reply:
Ok, checked and removed